parsec/core/gui/devices_widget.py

```diff
@@ -51,7 +51,7 @@ def show_context_menu(self, pos):
         menu.exec_(global_pos)
 
     def change_password(self):
-        self.change_password_clicked.emit(self.device_name)
+        self.change_password_clicked.emit(self.device_info.device_label)
```

Comment:
shouldn't it be `self.device_info.device_display` given `device_label` can be `None` ?

besides `self.change_password_clicked` is connected to `DevicesWidget.change_password` which make no use of it parameter `device_name`... so do we really need it ?

Reply:
max spots it 1 minute before you ;)